docs(examples): Add example for SegmentedControl with text buttons (#1623)
Workday/canvas-kit: modules/react/_examples/stories/SegmentedControlWithText.stories.mdx

import {Meta, Story} from '@storybook/addon-docs';
import {Basic} from './examples/SegmentControlWithText';

<Meta title="Examples/SegmentControl With Text/React" />

# Canvas Kit Examples

## SegmentControl With Text

> As in v7, IconButton was restructured, so SegmentControl was changed to support new buttons with

Reviewer: ```suggestion
As of `v7`, IconButton was removed, so SegmentControl was changed to support new buttons with
```

> icon only. This example helps to upgrade SegmentControl but still use text inside buttons until CK

Reviewer: ```suggestion
icon only. This example shows how you use SegmentControl with text until CK
```

> starts support text in SegmentControl buttons.

Reviewer: ```suggestion
starts supporting text in SegmentControl buttons. If you'd like to follow along, we have an [open issue](https://github.com/Workday/canvas-kit/issues/207) to enhance the SegmentedControl
```
